terraform/rancher/project/kafka.tf

```diff
@@ -27,7 +27,7 @@ resource "rancher2_app_v2" "kafka" {
         enabled: true
         resources:
           limits:
-            memory: 512Mi
+            memory: 1280Mi
```

Comment:
the same as PSQL var implementation, just skipped :(

Reply:
Not sure